--- ardour-3/gtk2_ardour/ardour.appdata.xml.in.in	2019-02-07 06:42:07.636417275 -0500
+++ ardour-3/gtk2_ardour/ardour.appdata.xml.in.in.new	2019-02-07 08:06:36.733221723 -0500
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
-  <id>@ARDOUR_EXEC@.desktop</id>
+  <id>org.ardour.Ardour</id>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0+</project_license>
   <name>Ardour</name>
@@ -18,14 +18,16 @@
       composers.
     </p>
   </description>
+  <content_rating type="oars-1.1" />
+  <launchable type="desktop-id">org.ardour.Ardour.desktop</launchable>
   <url type="homepage">https://ardour.org</url>
   <screenshots>
       <screenshot type="default">
-        <image>https://community.ardour.org/files/images/ardour-editor-window-1600x900.png</image>
+        <image>https://ardour.org/images/big_editor4.png</image>
         <caption>The Editor window where you can arrange audio and MIDI data along a timeline</caption>
       </screenshot>
       <screenshot>
-        <image>https://community.ardour.org/files/images/ardour-mixer-window-1600x900.png</image>
+        <image>https://ardour.org/images/big_mixer4.png</image>
         <caption>The Mixer window which includes channel strips for each track and bus</caption>
       </screenshot>
   </screenshots>
